instagram/instagram/src/components/PostsContainer/PostsPage.js

```diff
@@ -0,0 +1,55 @@
+// -IMPORTS- // 
+    import React from 'react'
+
+
+    // COMPONENTS
+    import SearchBar from '../SearchBar/SearchBar'
+    import PostContainer from './PostContainer'
+
+    // CSS
+    import '../components.css'
+// -**-END-**-//
+// -Start Code- //
+function PostsPage(props) {
+
+        return (
+            <div className='PostsPage'>
+                <SearchBar 
+                    // Search Functionality
+                        searchText={props.searchText}
+                        filterSearchText={props.filterSearchText}
+                />
+
+                {console.log(props)}
+                {console.log(props.posts)}
+                {console.log(props.posts.length)}
+
+                {/* // is there any data in posts array?? */}
+                {props.posts.length > 0 ? (
+                    // YES - map the posts array
+                    props.posts.map( post => (
+                        // is there anything in the search bar?
+                        props.searchText !== '' ? (
+                            // YES - only send matched user name
+                            post.username === props.searchText ? (
```

Comment:
This change would result in a fuzzier search rather than an exact match. 
Ex: Searching for `z` would render philzcoffee
```suggestion
                            post.username.includes(props.searchText) ? (
```